### PR TITLE
WOR-1803 Part 2: Move group-common to groups folder

### PR DESCRIPTION
**Pull request by blakery**
## Summary of changes:
* Rename components from `User{Component}` to `Member{Component}`, to differentiate it from the global `User` concept, and to be consistent with usage in billing
* Combine `MemberCard` list with headers, in a combined table component, since they are always used the same way, and are conceptually linked
* Move components from common `components` directory, to a subdirectory in the `groups` directory 

### Testing strategy

- [ ]

### CODE REVIEW

File: src/billing/Members/Members.tsx [hunk 1]
```diff
@@ -1,57 +1,46 @@
...
+import { Member, MemberTable } from 'src/groups/Members/MemberTable';
+import { NewMemberModal } from 'src/groups/Members/NewMemberModal';
 import { Ajax } from 'src/libs/ajax';
 import { BillingRole } from 'src/libs/ajax/Billing';
 
 interface MembersProps {
   billingProjectName: string;
   isOwner: boolean;
-  projectUsers: User[];
+  projectUsers: Member[];
```
Comment (cahrens) [inline src/billing/Members/Members.tsx]: Do you want to go all the way and rename things like `projectUsers` to `projectMembers`?

Comment (blakery) [reply src/billing/Members/Members.tsx]: good idea

**Review by aherbst-broad — APPROVED** [review]

**Comment by cahrens** [comment]
Why is the font so large on the group name? It's this way on dev, but I'm pretty sure it didn't _used_ to be so big.

image

**Comment by blakery** [comment]
> Why is the font so large on the group name? It's this way on dev, but I'm pretty sure it didn't _used_ to be so big.
> 
> image

The group name cell had `Style.cardList.longTitle` added to its styles, which increased the font size. 

```
      <div role='rowheader' style={{ marginRight: '1rem', ...Style.noWrapEllipsis }}>
        <a
          href={isAdmin ? getLink('group', { groupName }) : undefined}
          aria-disabled={!isAdmin}
          style={{ ...Style.cardList.longTitle, color: isAdmin ? colors.accent() : undefined }}
        >
          {groupName}
        </a>
```

Aside from the font size, the rest of the styles from `Style.cardList.longTitle` are redundant, because they're included in `Style.noWrapEllipsis`.

I looked back, and those styles were present there at least a month ago - so it's not new. But it does make sense to remove them.

**Comment by cahrens** [comment]
Looks much better, thanks!

**Review by cahrens — APPROVED** [review]

**Pull request merged**